docker/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM rootproject/root:6.24.00-conda
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get -y install tzdata 
+RUN apt-get update  && apt-get -y install cmake vim git
+
+ENV HOME /home/ilc
+WORKDIR ${HOME}
+
+RUN conda create -n root_env root -c conda-forge 
+
+SHELL ["conda", "run", "-n", "root_env", "/bin/bash", "-c"]
+
+
+RUN git clone https://github.com/iLCSoft/LCIO.git \
+   && cd LCIO \
+   && git checkout v02-16-01 \ 
+   && mkdir build \
+   && cd build \
+   && cmake -DBUILD_ROOTDICT=ON -D CMAKE_CXX_STANDARD=17 ..  \
+   && make -j 8 install && cd .. ; rm -rf ./build
+
+
+WORKDIR ${HOME}
+
+## Delphes installation from local (unfortunate) due to TF1.h error in ROOT 6.22
+## details : https://cp3.irmp.ucl.ac.be/projects/delphes/ticket/1449
+
+RUN wget http://cp3.irmp.ucl.ac.be/downloads/Delphes-3.5.0.tar.gz \
```

Comment:
Could we use the latest (i.e. current master) of delphes for this? See discussion in: #65